content/blog/2022/10/envvar-policy.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Enforcing compliance in your environment variables
+authors:
+- José Guilherme Vanz
+date: 2022-11-01
+---
+
+We're glad to present to Kubewarden users the new [environment-variable-policy](https://github.com/kubewarden/environment-variable-policy). It's able to inspect containers, init containers, and ephemeral containers, and restricts their usage of environment variables by checking their names and values defined under the containers' [`env[*]`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core).
+
+As always, the policy can be found in [Github](https://github.com/kubewarden/environment-variable-policy) and all the  artifacts, including the BOM files, are signed with Sigstore.
```

Comment:
I would replace the GitHub link with a link to ArtifactHub, right to the policy page